com.github.emmanueltouzery.hotwire.json

```diff
@@ -0,0 +1,129 @@
+{
+  "app-id": "com.github.emmanueltouzery.hotwire",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "20.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "hotwire",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
```

Comment:
```suggestion
```

Reply:
Usually if the app supports wayland, it doesn't have x11, just fallback-x11 (which is only available if there is no wayland available)